src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxFactoryTests.cs

```diff
@@ -554,6 +554,14 @@ public void TestSpacingOnCoalescing()
             Assert.Equal("x is object ?? y", syntaxNode3.ToFullString());
         }
 
+        [Fact]
+        [WorkItem(70135, "https://github.com/dotnet/roslyn/issues/70135")]
+        public void TestSpacingNumeric()
```

Comment:
Move the test to `SyntaxNormalizerTests.cs` file

Reply:
done